src/tsickle.ts

```diff
@@ -666,16 +684,41 @@ class Annotator extends ClosureRewriter {
       this.generatedExports.add(name);
       reexports.add(name);
     }
-
     return toArray(reexports.keys());
   }
 
   /**
+   * Write an `exports.` assignment for each type exported in named exports.
+   * TypeScript by itself does not export non-value symbols (e.g. interfaces, typedefs), as it
+   * expects to remove those entirely for runtime. For Closure, types must be
+   * exported as downstream code will import the type.
+   *
+   * The tsickle pass turns interfaces into values by generating a `function MyInterface() {}` for
+   * them, so in the second conversion pass, TypeScript does export a value for them. However for
+   * pure typedefs, tsickle only generates a property access with a JSDoc comment, so they need to
+   * exported explicitly here.
+   */
+  private emitTypeDefExports(exports: ts.NamedExports) {
+    const typeChecker = this.program.getTypeChecker();
+    for (const e of exports.elements) {
+      let sym = typeChecker.getSymbolAtLocation(e.name);
+      if (sym.flags & ts.SymbolFlags.Alias) sym = typeChecker.getAliasedSymbol(sym);
+      const aliased = this.symbolsToAliasedNames.get(sym);
+      const isTypeAlias = sym.flags & ts.SymbolFlags.TypeAlias;
+      const name = getIdentifierText(e.name);
+      // console.log('exported:', name, typeTranslator.symbolToDebugString(sym));
+      if (!isTypeAlias) continue;
+      const typeName = aliased || name;
+      this.emit(`\n/** @typdef {${typeName}} */\nexports.${name};`);
```

Comment:
Hahaha. So it turns out, if you write:

```
/** @mopskotze {number} */
exports.Foo;
```

... that actually declares a property `Foo` on the export object, with an unknown type. Same for `@typdef`. So our Closure integration test didn't fail. In g3, it seems like we error on unknown JSDoc tags, but maybe not for comments that could just be comments in the file?

In any case, fixed.